### PR TITLE
Invalid Records UI

### PR DESCRIPTION
**Pull request by krishnaglick**
## Change description

> Additional Invalid processing work to function with Import button.

## Related issues

Does this PR fix a Github Issue?

> 

### Development

- [X] Application changes have been tested appropriately

### Impact

- [X] Code follows company security practices and guidelines
- [X] Security impact of change has been considered
- [X] Performance impact of change has been considered
- [X] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

>

### Code review

- [X] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [X] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: core/__tests__/actions/records/importAndExport.ts [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
 } from "../../../src";
-import { Op } from "sequelize";
+import {
+  RecordCreate,
+  RecordExport,
+  RecordImport,
+} from "../../../src/actions/records";
+import { SessionCreate } from "../../../src/actions/session";
+
+var mockRecordResponseData: ReturnType<typeof jest.fn>;
```
Comment (krishnaglick) [inline core/__tests__/actions/records/importAndExport.ts]: Sometimes you need `var` for that global scope.

File: plugins/@grouparoo/spec-helper/src/lib/specHelper.ts [hunk 2]
```diff
@@ -383,15 +383,15 @@ export namespace helper {
...
+              return helper.recordResponseData(record, property.key);
             },
             recordProperties: async ({ properties, records }) => {
               const response = {};
               for (const record of records) {
                 response[record.id] = {};
                 for (const property of properties) {
                   response[record.id][property.id] = [
-                    recordResponseData(record, property.key),
+                    helper.recordResponseData(record, property.key),
```
Comment (krishnaglick) [inline plugins/@grouparoo/spec-helper/src/lib/specHelper.ts]: Needed explicit referencing of `helper` so I can intercept this value in tests.

File: core/src/actions/records.ts [hunk 3]
```diff
@@ -173,6 +173,7 @@ export class RecordImport extends CLSAction {
     await record.buildNullProperties();
     await record.markPending();
     await record.import();
+    await RecordOps.computeRecordsValidity([record]);
```
Comment (evantahler) [inline core/src/actions/records.ts:176]: This probably should happen after the reload

Comment (krishnaglick) [reply core/src/actions/records.ts:176]: This causes the tests to fail, but the app seems fine. I'll need to look around to see why.

**Review by evantahler — APPROVED** [review]
Nice and clear changes

**Pull request merged**